reframe/frontend/cli.py

```diff
@@ -27,23 +27,15 @@
 def format_check(check, detailed):
     lines = ['  * %s (found in %s)' % (check.name,
                                        inspect.getfile(type(check)))]
+    flex = 'flexible' if check.num_tasks <= 0 else 'inflexible'
```

Comment:
```python
    flex = 'flexible' if check.num_tasks <= 0 else 'standard'
```

Reply:
Or should we just add sth to the output if only the test is flexible?

Reply:
I think it's better to print something in any case.